### ISSUE TITLE
Mouse Cursor Tiny

### ISSUE THREAD

**Issue by norpol** [swaywm/sway#4610]
My mouse cursor is super tiny in wayland applications. The scaling in swaybar and xwayland applications is correct though. On the screenshot you can see firefox-wayland (from nixos) but this also happens in other wayland applications such as eye-of-gnome or termite.

As suggested elsewhere I've already added dmz-black theme to `~/.icons/default`, this only changed the cursor theme for swaybar though. 

* Sway Version: `sway version da0ad4c39ed91c0d485b3d519ad1631ea6a46ba2`

* Configuration File:

```
output eDP-1 {
        pos 0,0 res 1920x1080 scale 1.25
        bg ~/static/pictures/wallpapers/subway.jpg stretch
}
```

* Screenshots:

![file2](https://user-images.githubusercontent.com/2230917/66200087-89a5a180-e6a0-11e9-8a74-958a71982a61.png)
![file](https://user-images.githubusercontent.com/2230917/66200088-89a5a180-e6a0-11e9-8a38-4f13457f61cd.png)

**Comment by dnkl** [comment]
There isn't (yet) a centralized, or standardized way to tell Wayland applications which xcursor theme (and at which size) they should be using.

What Sway does is set and export the environment variables `XCURSOR_THEME` and `XCURSOR_SIZE`. Some applications look at these (for example, swaybar). XWayland is a special case, handled by Sway.

**Comment by progandy** [comment]
firefox and gtk applications are probably using the gtk/gnome settings:

https://github.com/swaywm/sway/wiki/GTK-3-settings-on-Wayland

**Comment by hypergig** [comment]
The gnome tweak tool does work in sway, if you are (or have to be) on ubuntu
```bash
sudo apt install gnome-tweak-tool
```
you can fix your appearance setting pretty quickly there, then hardcode them in config with the ^ above guides.

What happened me to was, I started gnome in wayland to help troubleshoot a zoom issue... it must have blew out all my gtk settings on first start... pretty frustrating.

**Comment by eoli3n** [comment]
Same problem here

**Comment by emersion** [comment]
This is a GTK bug. They should handle `XCURSOR_THEME` and `XCURSOR_SIZE`.

**Comment by mrusme** [comment]
I've stumbled into this as well and I cannot seem to get it fixed. Every application (GTK, as well as Qt5 ones) respect the settings that I have. However, parts of Sway do not.

```sh
 ▲ ~ echo $XCURSOR_THEME $XCURSOR_SIZE
Vimix-White 32
 ▲ ~ gsettings get org.gnome.desktop.interface cursor-theme
'Vimix-White'
 ▲ ~ gsettings get org.gnome.desktop.interface cursor-size
32
 ▲ ~ /bin/cat .icons/default/index.theme
[icon theme]
Inherits=Vimix-White
```

GTK, as well as Qt applications render the cursor with the correct size (32). As soon as I move the mouse on the gaps between windows in Sway, it gets smaller (~24).

I tried following the conversations and patches on the wlroots list, but I didn't quite figure out what the status is. Is there a D-BUS implementation available now and how would it need to be configured?

Thank you!
